src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,94 @@
+import React,{ useEffect } from 'react'
+import { Navbar } from "../../Components/Navbar/Navbar"
+import { GenreCard } from '../../Components/GenreCards/GenreCard'
+import LibraryIllustration from "../..//Assets/Images/Library_Illustration_1.jpg"
+import './Home.css'
+import { newArrivalsProductList } from "../../Components/NewArrivals/new-arrivals"
+import { ProductCard } from "../../Components/Card/ProductCard"
+import { Footer } from "../../Components/Footer/Footer"
+import { Link } from "react-router-dom"
+import { useProductAvailable } from "../../Context/product-context"
+import { useGenre } from "../../Context/genre-context"
+import { useLocation } from "react-router-dom"
+
```

Comment:
fix: You can refractor all your imports in a single file. This would improve readability

Reply:
Done.